chingu-voyages/v51-tier3-team-33
# Feature/create navbar for the dashboard page

# Description
Created UI components for the navbar and basic page structure in the app

## Main changes explained:

- Updated pages structure:
![Screenshot 2024-09-18 at 10 29 05 AM](https://github.com/user-attachments/assets/42c18c42-dc46-4ea6-bb61-a16406f96b0f)

- Created UI components for the navbar (sidebar):

![Screenshot 2024-09-18 at 10 29 35 AM](https://github.com/user-attachments/assets/a894f823-aa82-4dd1-8d80-0ae412c4ae91)


-Added colors from Figma to tailwind.config.ts


## How to test:
1. check into current branch
2. do `npm install` and `npm run dev` to run this PR locally
3. go to `/home` 
4. verify you see the navbar on the side of the page
5. click on the nav links in the top left hand corner and verify that you are directed to an appropriate page (there is no pages content atm but you will see the url changing and a heading changing in the middle of the main section.

## Screenshots or videos of changes:

![Screenshot 2024-09-16 at 3 38 59 PM](https://github.com/user-attachments/assets/e52639b1-9540-4b7a-89ef-ecc1959e8089)


## Note:
Please disregard the messy tailwind.config.ts file. I will be working on updating it.
